packetbeat/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Beats dockerfile used for testing
+FROM ruflin/beat-tester
+MAINTAINER Nicolas Ruflin <ruflin@elastic.co>
+
+RUN set -x && \
+    apt-get update && \
+    apt-get install -y netcat python-virtualenv python-pip && \
+    apt-get clean
+
+## Install go package dependencies
+RUN set -x \
+  go get \
+	github.com/pierrre/gotestcover \
+	github.com/tsg/goautotest \
+	golang.org/x/tools/cmd/cover \
+	golang.org/x/tools/cmd/vet
+
+ENV GO15VENDOREXPERIMENT=1
+ENV PYTHON_ENV=/tmp/python-env
+
+
+RUN test -d ${PYTHON_ENV} || virtualenv ${PYTHON_ENV}
+RUN . ${PYTHON_ENV}/bin/activate && pip install nose jinja2 PyYAML nose-timer
+
+RUN apt-get install -y libpcap-dev geoip-database && apt-get clean
```

Comment:
Wouldn't this build faster and more space efficient (reduced number of layers) if libpcap-dev and geoip-database where installed up on line 7.